src/forms/FormFieldText/FormFieldTextTypes.tsx

```diff
@@ -25,4 +25,14 @@ export type TextFieldDef = {
 	 * input type.
 	 */
 	type?: string;
+	/**
+	 * Minimum number of rows to display when multiline option
+	 * is set to true.
+	 */
+	minRows?: string | number;

```

Comment:
I know MUI's documentation says it can be string or number, but I believe we should keep it 100% as number (for both min and max).
```suggestion
	minRows?: number;

```